Tests/DatadogTests/Datadog/RUMMonitorTests.swift

```diff
@@ -1422,10 +1423,6 @@ class RUMMonitorTests: XCTestCase {
             Make sure `Global.rum = RUMMonitor.initialize()` is called before any action happens.
             """
         )
-
-        URLSessionAutoInstrumentation.instance?.swizzler.unswizzle()
-        rumInstrumentation.viewControllerSwizzler?.unswizzle()
-        rumInstrumentation.userActionsAutoInstrumentation?.swizzler.unswizzle()
```

Comment:
We don't need it because it is part of deinitialization, right?
```swift
Datadog.flushAndDeinitialize()
```

Reply:
Yes exactly!